flink-connector-mongodb/src/test/java/org/apache/flink/connector/mongodb/sink/MongoSinkITCase.java

```diff
@@ -101,9 +102,11 @@ void testWriteToMongoWithDeliveryGuarantee(DeliveryGuarantee deliveryGuarantee)
             throws Exception {
         final String collection = "test-sink-with-delivery-" + deliveryGuarantee;
         final MongoSink<Document> sink = createSink(collection, deliveryGuarantee);
-        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        Configuration config = new Configuration();
+        config.set(RestartStrategyOptions.RESTART_STRATEGY, "disable");
```

Comment:
nit: seems `RestartStrategyUtils.configureNoRestartStrategy(env)` is the standard way to set restart strategy since Flink 2.0.